test/CheckoutSdk.Tests/Payments/GetPaymentTests.cs

```diff
@@ -184,6 +184,7 @@ public async Task CanGetPaymentAction()
             paymentAction.AuthCode.ShouldBe(paymentResponse.Payment.AuthCode);
             paymentAction.Type.ShouldBe(ActionType.Authorization);
             paymentAction.Links.ShouldNotBeNull();
+            paymentAction.Approved.ShouldNotBeNull();
```

Comment:
@philippe-leonhardt-cko `Approved` is a boolean field so it will never be null. This test should test for `true` or `false` values